### DISCUSSION TITLE
@UseClasspathSqlLocator with custom path to SQL file

### DISCUSSION THREAD

**Discussion by wKoja** [jdbi/jdbi#2202] [Q&A] [answered]
Is there a way to tell JDBI to use a custom path with this annotation instead of having to make it match the interface's fully qualified name? Similar to how ```ClasspathSqlLocator.create().locate('custom/path/to/custom-query)``` works.

**Comment by hgschmie** [comment, marked as answer]
Unfortunately not. Right now, it only supports the matching to the fully qualified package name.

**Reply by wKoja** [reply]
Is this something that's planned for the future? If not, can I work on this? I really need this feature in the project I'm working on.

**Reply by stevenschlansker** [reply]
I don't think any contributor is currently working on this. We'd love to have your contribution.

**Reply by wKoja** [reply]
Alright, I'll give it a try then. Thank you all.
